docs/reference/method/MongoDBCollection-bulkWrite.txt

```diff
@@ -70,6 +70,15 @@ Parameters
        - If ``true``, allows the write operation to circumvent document level
          validation. Defaults to ``false``.
 
+     * - codec
+       - MongoDB\\Codec\\DocumentCodec
+       - .. include:: /includes/extracts/collection-option-codec.rst
+
+         The codec will be utilized for ``insertOne`` and ``replaceOne``
+         operations.
+
```

Comment:
Question:
Does the future tense "will" indicate that this is upcoming functionality? If it's current functionality, I think it could be clearer to use present tense and active voice (the [first two standards in the Style Guide quickstart](https://www.mongodb.com/docs/meta/style-guide/quickstart/)). 

E.g.:

"Bulk operations use the code for insertOne and replaceOne operations."

Reply:
This is all current functionality. I don't believe we speak about upcoming functionality anywhere in the PHPLIB docs.

Will change to present tense.